include/ignition/rendering/GpuRays.hh

```diff
@@ -199,7 +199,7 @@ namespace ignition
       public: virtual double VerticalResolution() const = 0;
 
       /// \brief Set the scanning pattern
-      public: virtual void SetScanningPattern(const ScanningPattern pattern) = 0;
+      public: virtual void SetScanningPattern(const ScanningPattern scanninPattern) = 0;
```

Comment:
You can call the parameter `_pattern` or `_scanningPattern`. A `_` prefix for parameters is the norm in this codebase.